### DISCUSSION TITLE
A published GitHub Maven package is not available under the repository

### DISCUSSION THREAD

**Discussion by reugn** [community/community#7499] [Actions] [answered]
https://github.com/reugn/gravity/packages

**Comment by alexcnichols** [comment, marked as answer]
:wave: Hi @reugn - If you're looking for some help with publishing to the Apache Maven registry within GitHub Packages, I suggest this article: https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry
